src/libclipper/include/clipper/task_executor.hpp

```diff
@@ -479,42 +496,17 @@ class TaskExecutor {
         (*cur_model_metric).throughput_->mark(batch_size);
         (*cur_model_metric).num_predictions_->increment(batch_size);
         (*cur_model_metric).batch_size_->insert(batch_size);
+        (*cur_model_metric)
+            .latency_->insert(static_cast<int64_t>(task_latency_micros));
```

Comment:
Move this call inside the for loop (line 502). The model latency histogram should have an entry for every prediction request.

Reply:
Done